doc/task_breakdown.md

```diff
@@ -0,0 +1,54 @@
+# SQL Query Builder Task Breakdown
+
+## 1. Table Management
+- Add a list of tables from the database.
+- Add functionality for users to select one or more tables for the query.
+- Add functionality for users to remove tables from the selection.
+- Implement removal of related joins, filters, sorting and grouping options.
+- Implement dynamic updating of the lists of available and columns.
+---
+
+## 2. Connection Management
+- Implement automatic detection of possible joins between tables based on foreign keys.
+- Add functionality for users to manually configure joins by specifying conditions and types (`INNER JOIN`, `LEFT JOIN`, etc.).
+- Add functionality for users to modify or delete existing joins.
```

Comment:
again what to do if you add more than 1 table without joins, or delete all joins if you have more than 1 table, how it will work?